webots_ros2_epuck2/README.md

```diff
@@ -0,0 +1,85 @@
+## Run demo
+
+``` 
+ros2 launch webots_ros2_epuck2 example_launch.py
+```
+
+and you can use `teleop_twist_keyboard` in another window to control the robot:
+
+``` 
+ros2 run teleop_twist_keyboard teleop_twist_keyboard
+```
+
+but make sure you have `ros-eloquent-teleop-twist-keyboard` installed.
+
+### Topics Overview
+
+``` 
+$ ros2 topic list -t
+/camera/image_raw/camera_info [sensor_msgs/msg/CameraInfo]
+/camera/image_raw [sensor_msgs/msg/Image]
+/cmd_vel [geometry_msgs/msg/Twist]
+/distance/ps0 [sensor_msgs/msg/Range]
+/distance/ps1 [sensor_msgs/msg/Range]
+/distance/ps2 [sensor_msgs/msg/Range]
+/distance/ps3 [sensor_msgs/msg/Range]
+/distance/ps4 [sensor_msgs/msg/Range]
+/distance/ps5 [sensor_msgs/msg/Range]
+/distance/ps6 [sensor_msgs/msg/Range]
+/distance/ps7 [sensor_msgs/msg/Range]
+/distance/tof [sensor_msgs/msg/Range]
```

Comment:
It seems we are missing the ground sensors, if they are present (``E-puckGroundSensors`` node in the ``groundSensorsSlot`` field). But this can go in another PR of course.

Reply:
Added more detailed introduction